str_analysis/generate_gnomad_json.py

```diff
@@ -1,3 +1,95 @@
+                                     PcrProtocol : pcr_free
+                                        Genotype : 6/13
+                                         Allele1 : 6
+                                         Allele2 : 13
+                      GenotypeConfidenceInterval : 6-6/11-30
+                   GenotypeUsingOfftargetRegions : 6/13
+                    Allele1UsingOfftargetRegions : 6
+                    Allele2UsingOfftargetRegions : 13
+ GenotypeConfidenceIntervalUsingOfftargetRegions : 6-6/11-30
+                                 ReadvizFilename : c82034cf2aad813a07a8523898d64c81148.svg
+
+Id : This id is unique to each STR locus and repeat, meaning that it differs between repeats and their adjacent repeats
+    at a locus. For example the main GAA repeat at the FXN locus has id "FXN" while the adjacent poly-A repeat has id
+    "FXN_A". It corresponds to the "VariantId" field in the ExpansionHunter variant catalogs @ https://github.com/broadinstitute/str-analysis/tree/main/str_analysis/variant_catalogs
+LocusId: This id is unique to each STR locus. It corresponds to the "LocusId" field in the ExpansionHunter variant catalogs @ https://github.com/broadinstitute/str-analysis/tree/main/str_analysis/variant_catalogs
+    and can be used to look up reference information about each locus in the variant catalogs - including the
+    gene name, disease associations, mode of inheritance, and pathogenic threshold. For most but not all loci, the
+    LocusId is identical to the gene name that contains the locus.
+ReferenceRegion: Genomic interval deliniating the exact boundaries of the STR repeat in the reference genome.
+    The start coordinate is 0-based.
+Chrom: The chromosome of the ReferenceRegion. This is provided as a separate column for convenience.
+Start_0based: The 0-based start coordinate of the ReferenceRegion. This is provided as a separate column for convenience.
+End: The end coordinate of the ReferenceRegion. This is provided as a separate column for convenience.
+Motif: The repeat unit of the STR locus. For example this would be "GAA" at the FXN locus.
+IsAdjacentRepeat: True or False depending on whether this row represents the main repeat at a locus or an adjacent repeat.
+    Adjacent repeats are included for some loci either for technical reasons to improve ExpansionHunter accuracy, or
+    due to research interest in the size of these adjacent repeats.
+Population: The gnomAD ancestry group of the individual. Possible values are: "afr", "ami", "amr", "asj", "eas", "fin",
+    "mid", "nfe", "oth", "sas"
+Sex: Possible values are "XX" for females and "XY" for males
+Age: The age of the individual at the time when they enrolled in one of the research studies underlying gnomAD.
+    The values represent 5 year bins such as "20-25" as well as ">80" for individuals over 80 years old and "<20" for
+    individuals younger than 20. For individuals with unknown age, the value is "age_not_available"
```

Comment:
I think for the main gnomAD release we leave this as a NULL value but this is fine for now